### ISSUE TITLE
Change commit message

### ISSUE THREAD

**Issue by alex27riva** [haydenull/logseq-plugin-git#13]
Hi, is possible to change the default commit message?
maybe through a setting?

Bye

**Comment by goldan** [comment]
This would be helpful!

**Comment by ewindes** [comment]
Agree.  Would be useful for cases where I'm running logseq on multiple hosts, and would like to add hostnames to the git commit messages.

**Comment by k2s** [comment]
Pull request https://github.com/haydenull/logseq-plugin-git/pull/39 removes the commit message for now.

Time and author are part of commit anyway. That it was committed by this plugin? Well, the message is empty :-).

Logseq doesn't seam to provide hostname in its API. This plugin can't access your machine outside the boundaries given by Logseq API.

Could you, please, provide other useful ideas for commit message?

**Comment by sdasda7777** [comment]
Would it be possible to add an option to show confirmation prompt with option to enter commit message on Ctrl+S?

**Comment by naught101** [comment]
> Would it be possible to add an option to show confirmation prompt with option to enter commit message on Ctrl+S?

Yes, this would be extremely valuable. Meaningful git messages can be very helpful when reviewing history.
